src/utils/scenarioParameters/ScenarioParametersUtils.js

```diff
@@ -118,41 +109,36 @@ const _getVarTypeDefaultValue = (varType, subType) => {
   return VAR_TYPES_DEFAULT_VALUES[varType];
 };
 
-const _getDefaultParameterValueFromConfig = (parameterId, configParameters) => {
-  return configParameters?.[parameterId]?.defaultValue;
-};
-
 const _findParameterInSolutionParametersById = (parameterId, solutionParameters) => {
   return solutionParameters?.find((param) => param.id === parameterId);
 };
 
-function _getDefaultParameterValueFromDefaultValues(parameterId, configParameters, parameterVarType) {
-  const subType = ConfigUtils.getParameterSubType(parameterId, configParameters);
+function _getDefaultParameterValueFromDefaultValues(solutionParameter, parameterVarType) {
+  const subType = ConfigUtils.getParameterAttribute(solutionParameter, 'subType');
   return _getVarTypeDefaultValue(parameterVarType, subType);
 }
 
-const _getDefaultParameterValue = (parameterId, solutionParameters, configParameters) => {
-  let defaultValue = _getDefaultParameterValueFromConfig(parameterId, configParameters);
-  if (defaultValue !== undefined) {
-    return defaultValue;
-  }
-
+const _getDefaultParameterValue = (parameterId, solutionParameters) => {
   const solutionParameter = _findParameterInSolutionParametersById(parameterId, solutionParameters);
   if (!solutionParameter) {
     console.warn(`Unknown scenario parameter "${parameterId}"`);
     return undefined;
   }
-  defaultValue = solutionParameter.defaultValue;
 
-  if (defaultValue === null) {
+  let defaultValue = solutionParameter.defaultValue;
+  // defaultValue might not be in parameter data for parameters overridden by local config; when sent by the back-end,
+  // parameters should always have a defaultValue property, that will be set to 'null' by default
+  if ('defaultValue' in solutionParameter === false || defaultValue === null) {
```

Comment:
Just a suggestion: we can symplify `'defaultValue' in solutionParameter === false` to` (!('defaultValue' in solutionParameter)` :slightly_smiling_face: